model/flow/quorum_certificate.go

```diff
@@ -25,3 +25,13 @@ type QuorumCertificate struct {
 	// from all signers.
 	SigData []byte
 }
+
+// QuorumCertificateWithSignerIDs is a QuorumCertificate model with signer IDs instead of indices.
+// it is needed for backward-compatibility with the FlowEpoch smart contract which generates QCs
+// using signer IDs.
```

Comment:
```suggestion
// QuorumCertificateWithSignerIDs is a QuorumCertificat, where the signing nodes are identified via their `flow.Identifier`s instead of indices. Working with IDs as opposed to 
// indices is less efficient, but simpler, because we don't require a canonical node order. 
// It is used for bootstrapping new Epochs, because the FlowEpoch smart contract has no
// notion of node ordering. 
```